src/index.ts

```diff
@@ -165,92 +167,145 @@ app.use('/', async (c) => {
 
 	console.log("processing messages")
 	const msgResps = await Promise.all(messagesQ.map(async (message) => {
+		// if the demo is turned on do something
 		if (message.message && demoSwitch) {
-			console.log("doing geolookup for ", message.message)
-			// if message follows format then try and create ping out of it
-			const elems = message.message.text.split(".").filter(element => element.replace(" ","").length > 0)
-			console.log(elems)
-			if (elems.length > 1) {
-				const msg = elems[0]
-				const locq = elems[1]
-				console.log("doing geo query")
-				const resp = await fetch('https://maptoolkit.p.rapidapi.com/geocode/search?' + new URLSearchParams({
-					q: locq,
-					countrycodes: 'TW,US',
-					language: 'en',
-					limit: '1'
-				}),
-					{
-						method: "GET",
-						headers: {
-							'X-RapidAPI-Key': c.env.RAPID_API_KEY,
-							'X-RapidAPI-Host': 'maptoolkit.p.rapidapi.com'
+			// add routine to check video upload
+
+			const [pingSelector, videoSelector] = ["ping:", "video:"]
```

Comment:
I'm not liking this, would like to consider how to make more dynamic. et's add this req for the user in the README